iam-login-service/src/main/resources/application.yml

```diff
@@ -219,6 +219,9 @@ client-registration:
     default-id-token-validity-seconds: ${DEFAULT_ID_TOKEN_VALIDITY_SECONDS:600}
     default-refresh-token-validity-seconds: ${DEFAULT_REFRESH_TOKEN_VALIDITY_SECONDS:2592000}
 
+client:
+  track-last-used: ${IAM_CLIENT_TRACK_LAST_USED:false}
```

Comment:
I would force new IAM deployments to track the last used client (setting this property to `true` by default), but let's see what other colleagues think about it.

Reply:
Since you ask... Why should this feature be configurable?

Reply:
Tracking the `last_used` date will write more often to the DB, and may cause slowdowns in issuing tokens, especially when many different clients are used. Disabling this feature can be helpful when high throughput is required.

On small/simple IAM instances this should not be a problem, so I agree that the default could be set to ``true``. Currently I set it to ``false`` since it's the backward-compatible behavior.

Reply:
Make sense. On the other hand, the feature becomes useful exactly when there are many clients, because it helps with the cleanup.

Since we don't know the impact on performance, it may make sense to be able to enable/disable it, at least for the time being. What about keeping it as an undocumented option?

Reply:
I change the default to ``true``.